datadog_checks_base/tests/base/utils/test_http.py

```diff
@@ -95,6 +96,39 @@ def test_config_init_config_override(self):
         assert http.options['timeout'] == (16, 16)
 
 
+class TestRequestSize:
```

Comment:
Just curious, what do these numbers (i.e. `16384` and `512`) mean?

Reply:
`request_size` as bytes